src/screencast_keys/preferences.py

```diff
@@ -254,6 +254,12 @@ class SK_Preferences(bpy.types.AddonPreferences):
         default=True,
     )
 
+    load_on_startup = bpy.props.BoolProperty(
+        name="Load On Startup",
+        description="""(Experimental) Automatically enable screencast when blender is starting up""",
```

Comment:
Should this option experimental status?
If so, what is your concern?

Reply:
I added it here because I don't know your pipeline about introducing new features. If you think it must not be there, I'll move it to the common section.

Reply:
Moved `load_on_startup` to UI section